apps/condo/bin/create-billing-app.js

```diff
@@ -0,0 +1,70 @@
+const path = require('path')
+
+const faker = require('faker')
+
+const { prepareKeystoneExpressApp } = require('@open-condo/keystone/test.utils')
+
+const { BillingIntegration, BillingIntegrationAccessRight } = require('@condo/domains/billing/utils/serverSchema')
+const { User } = require('@condo/domains/user/utils/serverSchema')
+
+function getJson (data) {
+    try {
+        return JSON.parse(data)
+    } catch (e) {
+        return undefined
+    }
+}
+
+async function main (args) {
+    const [email, name, options] = args
+    let optionsJson = getJson(options)
```

Comment:
better to parse json in one place

const options = parseOptions(options)

or even better

const { contextDefaultStatus, category, dv, sender, billingIntegration, ... } = parseOptions(options)